hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/ProtobufLogReader.java

```diff
@@ -410,8 +410,7 @@ protected boolean readNext(Entry entry) throws IOException {
       if (!walKey.hasFollowingKvCount() || 0 == walKey.getFollowingKvCount()) {
         LOG.trace("WALKey has no KVs that follow it; trying the next one. current offset={}",
           this.inputStream.getPos());
```

Comment:
Wow, that's a big one. When I saw dataloss explanation, I wanted to check for logs `WALKey has no KVs that follow it; trying the next one.` to see how many, if at all num of such references we see but then realized it is at TRACE level. Shall we make it at least DEBUG level?

Reply:
I'm OK with changing it to DEBUG, can file a new issue?

Reply:
Sounds good

Reply:
Oh I thought this was a PR which has already been merged so I said we can file a new issue for this...
Let me change the log level and update the PR...